osf/models/metaschema.py

```diff
@@ -75,7 +75,27 @@ def validate_metadata(self, metadata, reviewer=False, required_fields=False):
         try:
             jsonschema.validate(metadata, schema)
         except jsonschema.ValidationError as e:
-            raise ValidationValueError(e.message)
+            for page in self.schema['pages']:
+                for question in page['questions']:
+                    if e.relative_schema_path[0] == 'required':
+                        raise ValidationError(
+                            'For your registration the \'{}\' field is required'.format(question['title'])
+                        )
+                    elif e.relative_schema_path[0] == 'additionalProperties':
+                        raise ValidationError(
+                            'For your registration the \'{}\' field is extraneous and not permitted in your response.'.format(question['qid'])
+                        )
+                    elif e.relative_path[0] == question['qid']:
+                        if 'options' in question:
+                            raise ValidationError(
+                                'For your registration the \'{}\' field is invalid, your response must be one of the provided options.'.format(
+                                    question['title'],
+                                ),
+                            )
+                        raise ValidationError(
+                            'For your registration the \'{}\' field is invalid.'.format(question['title']),
```

Comment:
Ditto to the above I think?

Reply:
agreed "Your response to the the \'{}\' field is invalid." or similar